src/Data/DisjointMap.hs

```diff
@@ -395,6 +384,7 @@ name as they would call it. We begin with a rating data type:
 
 >>> import Data.Function ((&))
 >>> data Rating = Lowest | Low | Medium | High | Highest deriving (Eq,Ord,Show)
+>>> instance Semigroup Rating where (<>) = min -- needed for Monoid Rating
```

Comment:
The comment "needed for Monoid Rating" does not provide any information since `Semigroup` is a superclass of `Monoid`.